draft-ietf-moq-transport.md

```diff
@@ -2687,24 +2879,25 @@ Violation.
 * Parameters: The parameters are defined in {{version-specific-params}}.
 
 The publisher will respond with SUBSCRIBE_ANNOUNCES_OK or
-SUBSCRIBE_ANNOUNCES_ERROR.  If the SUBSCRIBE_ANNOUNCES is successful,
-the publisher will forward any matching ANNOUNCE messages to the subscriber
-that it has not yet sent.  If the set of matching ANNOUNCE messages changes, the
-publisher sends the corresponding ANNOUNCE or UNANNOUNCE message.
+SUBSCRIBE_ANNOUNCES_ERROR.  If the SUBSCRIBE_ANNOUNCES is successful, the
+publisher will forward any matching ANNOUNCE and PUBLISH messages to the
```

Comment:
Make it clear that existing ANNOUNCE / subscriptions are forwarded to new SUBSCRIBE_ANNOUNCES